tests/tools/hoppy/test_parser.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: UTF-8 -*-
+import mock
+import unittest
+
+from hamcrest import assert_that, has_entry, equal_to
+
+from ptp.tools.hoppy.parser import HoppyParser
+
+class TestHoppyParser(unittest.TestCase):
+
+    ###
+    # HoppyParser.is_mine
+    ###
+    def test_parser_hoppy_is_mine(self):
+        self.assertTrue(HoppyParser.is_mine('./'))
```

Comment:
Could you apply what the other tests are doing? Meaning to save the report in a python file and import it here. Also mock the underlying function that will read the report (see https://github.com/owtf/ptp/blob/develop/tests/tools/arachni/test_parser.py for instance).

Using relative directory `./` could introduce some bugs that would be annoying to find and fix.